### PR TITLE
Colorpicker, fix behavior for one word triggers for 2675

### PR DESCRIPTION
**Pull request by jkrobles**
Fixes: #2675 
Added JS to make single words work.

http://duck.co/ia/view/color_picker
Maintainer: @chrisharrill

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by zekiel** [comment]
Thanks @jkrobles  -- much appreciated! Someone will review this asap but there are a few things ahead of this PR that we need to get to. As a next step, we've invited you to our Contributors team (you should've gotten an invite to your Github notifications or email). You should also be able to accept the invite here: https://github.com/duckduckgo 

This will definitely be helpful for our [Programming Mission](https://duck.co/blog/post/297/help-for-programmers) and we'd generally love more help there. If you'd like to oversee the improvement of results for a particular language, feel free to edit the wiki table and jump in via the instructions at the top!

File: share/spice/color_picker/color_picker.js [hunk 1]
```diff
@@ -578,6 +578,8 @@
             //  format it for later processing. The result will have all spaces, and parentheses
             //  replaced with commas such that there will only be one comma between any text.
             //  For example, HSV(1, 2, 3) becomes hsv,1,2,3
+            query = query.replace('colourpicker','color picker');
+            query = query.replace('colorpicker','color picker');
             var possible_color_query = query.split(/[\s,()]+/).slice(2)
```
Comment (moollaza) [inline share/spice/color_picker/color_picker.js]: @jkrobles I think a more robust solution would be to use a single regex to replace `colou?r\s?picker` with ''?

**Comment by moollaza** [comment]
@jkrobles Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by akanshgulati** [comment]
@moollaza wanted to know the status on this, I can see the issue is still open. I think the regex will be better in this case as user can also search with "color_picker". We can later extend it to color chooser or synonyms too.

**Pull request closed without merging**